src/platform/utilities/registry-helpers.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Gets the root URL of an app from the registry in the content-build.
+ *
+ * @param {string} entryName - An app's entry name.
+ * @returns {string} An app's root URL.
+ */
+export const getAppUrl = entryName => {
+  // Global app registry defined in the Webpack config
+  const appRegistry = __REGISTRY__;
+
+  // Return empty string when registry is empty or not defined
+  if (!appRegistry.length) return '';
```

Comment:
Is there a case where it's expected that the registry isn't defined? Wondering if this should throw instead of just returning

Reply:
Great question, the registry isn't defined in the mocha setup for unit tests, so this prevents an error being thrown when running unit tests. Well technically it is, but it's an empty array

Reply:
Didn't know that! There's always something new to learn in this codebase, haha

Reply:
Haha, I had to add that into the mocha setup in this PR because I was running into problems with unit tests throwing an error because `__REGISTRY__` wasn't defined